packages/synapse-react-client/src/components/ColumnFilter.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import { useQueryContext } from './QueryContext/QueryContext'
+import Select, {
+  components,
+  ControlProps,
+  GroupBase,
+  MultiValue,
+} from 'react-select'
+import {
+  ColumnMultiValueFunction,
+  ColumnMultiValueFunctionQueryFilter,
+  FacetColumnResultValues,
+  QueryFilter,
+} from '@sage-bionetworks/synapse-types'
+import { useQuery } from '@tanstack/react-query'
+
+export type FilterProps = {
+  filterColumnName?: string
+}
+
+export const Control = ({
+  children,
+  ...rest
+}: ControlProps<any, boolean, GroupBase<any>>) => {
+  return (
+    <components.Control {...rest} className="form-control">
```

Comment:
Could you convert this to use MUI's `Autocomplete`  instead of using Bootstrap 4 + react-select? In general we are trying to migrate away from Bootstrap, and while we still use react-select, prefer to use MUI when possible.